stable/external-dns/values.yaml

```diff
@@ -185,3 +185,6 @@ crd:
   create: false
   apiversion: ""
   kind: ""
+
+ # Load balancer service to be used; ie: custom-istio-namespace/custom-istio-ingressgateway. Omit to use the default (istio-system/istio-ingressgateway)
```

Comment:
It seems you included a space before the comment:

```diff
- # Load balancer service to be used; ie: custom-istio-namespace/custom-istio-ingressgateway. Omit to use the default (istio-system/istio-ingressgateway)
+# Load balancer service to be used; ie: custom-istio-namespace/custom-istio-ingressgateway. Omit to use the default (istio-system/istio-ingressgateway)
```

Reply:
@juan131 thanks i have fixed spacing